README.md

```diff
@@ -1 +1,15 @@
-TODO
+# Amazon ECS CNI Plugins
+
+[![Build Status](https://travis-ci.org/aws/amazon-ecs-cni-plugins.svg?branch=dev)](https://travis-ci.org/aws/amazon-ecs-cni-plugins)
```

Comment:
`dev` or `master`?  Maybe leave it as `dev` now and change to `master` right before release?